main.py

```diff
@@ -169,6 +169,42 @@ def markers():
                                       is_thin),
                         mimetype="application/json")
 
+
+@app.route("/charts-data", methods=["GET"])
+@user_optional
+def charts_data():
+    logging.debug('getting charts data')
+    kwargs = get_kwargs()
+    accidents_vehicles_involved = Marker.bounding_box_query(is_thin=False, yield_per=50, involved_and_vehicles=True, **kwargs)
```

Comment:
you can do 

    accidents, vehicles, involved =... 

To get three lists and you won't need to use indices. 

Also, I think you can do these loops with list comprehension, just need to replace the dictionary element assignment by a function.